docs/Activity.md

```diff
@@ -0,0 +1,385 @@
+passed to the adapter as well as for activities created by the adapter.
+
+The only supported activity type is a [WhatsApp message](#whatsapp-message-activity)
+(`type: "message", channelId: "whatsapp"`).
+
+
+## WhatsApp Message Activity
+
+The sender of the activity (`from`) must be defined. The channel account ID
+(`from.id`) must be set to the WhatsApp phone number of the sender. In case of
+channel activities, the channel account name (`from.name`) may be set to the
+display name of the sender.
+
+In case of channel activities, the recipient of the activity (`recipient`) is
+defined. The channel account ID (`recipient.id`) is set to the WhatsApp phone
+number of the recipient.
+
+In case of channel activities, the account ID of the conversation in which the
+activity exists (`conversation.id`) is set to the WhatsApp phone number of the
+sender, the account name (`conversation.name`) may be set to the display name
+of the sender and the account Is Group (`conversation.isGroup`) is set to
+`false`. In case of bot activities, the account ID (`conversation.id`) must be
+set to the WhatsApp phone number of the recipient.
+
+The prior activity to which the activity is a reply (`replyToId`), the metadata
+pertaining objects (`entities`), the text format (`textFormat`), the language
+code of the text content (`locale`), the instructions how to present
+attachments (`attachmentLayout`), the interactive actions that may be displayed
+to the user (`suggestedActions`), the time at which the activity should be
+considered to be expired (`expiration`), the relative importance
+(`importance`), the terms that language processing systems can listen for
+(`listenFor`) and the programmatic action accompanying the user request
+(`semanticAction`) are ignored.
+
+The text-to-speech content of the activity (`speak`), the indication of whether
+the sender is anticipating a response (`inputHint`) and the alternate delivery
+paths (`deliveryMode`) must not be defined.
+
+Only supported WhatsApp messages are [audio](#whatsapp-audio-message-activity),
+[document](#whatsapp-document-message-activity),
+[image](#whatsapp-image-message-activity),
+[sticker](#whatsapp-sticker-message-activity),
+[template](#whatsapp-template-message-activity),
+[text](#whatsapp-text-message-activity) and
+[video](#whatsapp-video-message-activity) messages.
+
+
+### WhatsApp Audio Message Activity
+
+The channel data of the activity (`channelData`) must be defined. A content
+type (`channelData.contentType`) must be set to `"audio"`. A message template
+(`channelData.template`) must not be defined.
+
+The text content of the activity (`text`) must not be defined.
+
+Exactly one attachment to the activity (`attachments`) must be defined. The
+content URL (`attachments[i].contentUrl`) must be defined. The content
+(`attachments[i].content`) and the thumbnail URL
+(`attachments[i].thumbnailUrl`) must not be defined.
+
+An example of a WhatsApp audio message activity looks like this:
```

Comment:
You are right, It should be :D . The article got there since I was playing with multiple versions. Maybe we can omit the word activity and keep just WhatsApp audio message example.

Reply:
Changed now: https://github.com/yet-another-radek/botbuilder-adapter-tyntec-whatsapp/compare/f66608b..9a5f449